installation/aws/VmClarity.cfn

```diff
@@ -229,21 +229,46 @@ Resources:
               content:
                 Fn::Sub:
                   - |
-                    AWS_REGION=${AWS::Region}
-                    SCANNER_AWS_REGION=${AWS::Region}
-                    AWS_SUBNET_ID=${VmClarityScannerSubnet}
-                    AWS_SECURITY_GROUP_ID=${VmClarityScannerSecurityGroup}
-                    AWS_INSTANCE_TYPE=${ScannerInstanceType}
-                    SCANNER_KEY_PAIR_NAME=${KeyName}
-                    AWS_JOB_IMAGE_ID=${JobImageID}
+                    ##
+                    ## Orchestrator configuration
+                    ##
+                    # Host for the VMClarity backend server
                     BACKEND_REST_HOST=__BACKEND_REST_HOST__
+                    # Port number for the VMClarity backend server
                     BACKEND_REST_PORT=8888
+                    # Container image for Scanner instance
                     SCANNER_CONTAINER_IMAGE=${ScannerContainerImage}
+                    # Trivy server address
                     TRIVY_SERVER_ADDRESS=http://__BACKEND_REST_HOST__:9992
+                    # Grype server address
                     GRYPE_SERVER_ADDRESS=__BACKEND_REST_HOST__:9991
-                    DELETE_JOB_POLICY=${AssetScanDeletePolicy}
+                    # FreshClam mirror URL
                     ALTERNATIVE_FRESHCLAM_MIRROR_URL=http://__BACKEND_REST_HOST__:1000/clamav
-                  - JobImageID: !FindInMap
+                    # Resource cleanup policy
+                    DELETE_JOB_POLICY=${AssetScanDeletePolicy}
+                    # Provider to use
+                    PROVIDER=aws
+
+                    ##
+                    ## Provider configuration
+                    ##
+                    # The AWS region where the provider is deployed
+                    AWS_REGION=${AWS::Region}
```

Comment:
Is this needed for the AWS client still?

Reply:
Yes. This is for the AWS client which might be different than the Scanner region.